code/modules/robotics/bot/medbot.dm

```diff
@@ -38,7 +44,33 @@
 	var/treatment_fire = "saline"
 	var/treatment_tox = "charcoal"
 	var/treatment_virus = "spaceacillin"
+	/// the stuff the bot injects when emagged
+	var/list/dangerous_stuff = list()
+	/// Set this to make the bot only inject all this crap
+	var/list/override_reagent = list()
+	/// They'll stop stop injecting that crap if the patient has the per-inject amount
+	var/override_reagent_limit_mult = 0.9
 	var/terrifying = 0 // for making the medbots all super fucked up
+	/// List of drugs that terrifying derelist bots will inject
+	var/static/list/terrifying_meds = list("formaldehyde" = 15,
+																				"ketamine" = 5,
+																				"pancuronium" = 5,
+																				"haloperidol" = 5,
+																				"atropine" = 30,
+																				"methamphetamine" = 20,
+																				"blood" = 20,
+																				"fog" = 20,
+																				"calomel" = 10,
+																				"ether" = 10,
+																				"mercury" = 5,
+																				"loose_screws" = 10,
+																				"salts1" = 10,
+																				"salmonella" = 10,
+																				"MRSA" = 10,
+																				"rat_venom" = 10,
+																				"green mucus" = 10,
+																				"mucus" = 10,
+																				"e.coli" = 10)
```

Comment:
Not really a fan of a lot of these, especially the disease reagents.

Reply:
Okay I changed up most of these, replaced all the disease chems with more dangerous medical-sounding chems.